test/checker/show_files_test.go

```diff
@@ -316,6 +334,8 @@ c.js
 			}`,
 			expected: `
 
+latest stable version: 3.0
+
 current version: 2.0
```

Comment:
I think the phrasing it a bit confusing. Instead of current version it's the latest version. Maybe we could make the version as stable for example: `latest version (stable):` and same with the lastest 10 versions.

Reply:
Maybe I could change `current` to `most recent`?